sdk/core/core-https/src/policies/bearerTokenAuthenticationPolicy.ts

```diff
@@ -34,26 +73,73 @@ export function bearerTokenAuthenticationPolicy(
 ): PipelinePolicy {
   const { credential, scopes } = options;
   const tokenCache: AccessTokenCache = new ExpiringAccessTokenCache();
-  async function getToken(tokenOptions: GetTokenOptions): Promise<string | undefined> {
+  const { prepareRequest, getChallenge = defaultGetChallenge, processChallenge } =
+    options.challenge || {};
```

Comment:
this suggestion is just an excuse to use new syntax 

```suggestion
    options.challenge ?? {};
```